src/libraries/System.Diagnostics.Tracing/tests/BasicEventSourceTest/TestsManifestGeneration.Etw.cs

```diff
@@ -54,6 +54,9 @@ public void Test_EventSource_EtwManifestGeneration()
                     Thread.Sleep(TimeSpan.FromSeconds(5));
 
                     tracesession.Flush();
```

Comment:
Is `Flush` asynchronous?

Reply:
The API itself isn't. The OS signal being received on the receiver end to flush the session and fully write the file is.

Reply:
So calling `Flush` just tells ETW to flush its buffers, and _that_ is asynchronous from the `Flush` call.  Got it.